api/src/test/java/uk/gov/dhsc/htbhf/claimant/ClaimantLifecycleIntegrationTests.java

```diff
@@ -456,6 +494,17 @@ private UUID applyForHealthyStartOverridingEligibilityForAPregnantWomanWithNoChi
         return applyForHealthyStart(newClaimDTO, NO_CHILDREN);
     }
 
+    private UUID applyForHealthyStartOverridingEligibilityForAPregnantTeenager(LocalDate expectedDeliveryDate)
```

Comment:
Can we rename the method to say under18 rather than Teenager as 18 & 19 year olds are teenagers. Same goes for the test method name.